src/lib/components/ProjectPage/NewProjectPage.tsx

```diff
@@ -1,27 +1,27 @@
 import { PageLayout } from '../PageLayout';
 import { DISPLAY_TEXTS } from '../../consts/displayTexts';
-import { IRoutesNames, PROJECT_TYPE_QUERY } from '../../consts/routes';
+import { ERoutesNames, PROJECT_TYPE_QUERY } from '../../consts/routes';
 import { IProjectPageProps } from './ProjectPage.types';
 import { APP_BREADCRUMB } from '@/lib/consts/breadcrumbs';
-import { PROJECT_DISPLAY_TEXTS, ProjectType } from '@/lib/consts/projects';
+import { PROJECT_DISPLAY_TEXTS, EProjectType } from '@/lib/consts/projects';
 import { Card } from '../commons/Card';
 import { ProjectForm } from '../ProjectForm';
 import { useRouter } from 'next/router';
 import { useModalContext } from '@/lib/context/ModalProvider/ModalProvider';
 import { IBreadcrumbProps } from '../PageLayout/Breadcrubms';
 
 export const NewProjectPage = ({ projectType }: IProjectPageProps) => {
-  const title = DISPLAY_TEXTS.he.routeNames[IRoutesNames.Project];
+  const title = DISPLAY_TEXTS.he.routeNames[ERoutesNames.Project];
   const router = useRouter();
   const { closeModal } = useModalContext();
   const addProjectText = PROJECT_DISPLAY_TEXTS.he.getAddNewText(projectType);
   const projectsTypeBreadCrumb: IBreadcrumbProps = {
     text: PROJECT_DISPLAY_TEXTS.he.projectTypes[projectType],
     id: 'projectType',
     navList: [
-      ProjectType.Residential,
-      ProjectType.Entrepreneurship,
-      ProjectType.PublicSpace,
+      projectType.Residential,
```

Comment:
this should be EProjectType ... 
we looping over the enum values (same as ObjectValues(EProjectType).map(....)) compare it with the current projectType value provided by props